app/components/primitive-select/template.hbs

```diff
@@ -1,8 +1,19 @@
-<select class="form-control" name={{name}} {{action 'update' on="change"}}>
-  {{#if prompt}}
-    <option disabled>{{prompt}}</option>
-  {{/if}}
-  {{#each items as |item|}}
-      <option>{{item}}</option>
-  {{/each}}
-</select>
+{{#if data-stripe}}
```

Comment:
I don't think you need this conditional at all, because `<select name={{name}} data-stripe={{data-stripe}}>`  if called without `name` will not include the `name` attribute and without `data-stripe` will not include the `data-stripe` attribute.